### ISSUE TITLE
Build the solution clarification Ex 2 Task 3 Step 5

### ISSUE THREAD

**Issue by hope77777** [MicrosoftLearning/MB-500-Microsoft-Dynamics-365-Finance-and-Operations-Apps-Developer#11]
# Lab/Demo: 04
#### Step: 05

Description of issue

Suggest adding an additional "**Task**"(Task 4 after Task 3 to build the solution). 
Otherwise, I would make an additional step (**Step 6**) for "**Build the Solution**".
It's easy to miss that instruction otherwise. 

Repro steps:
https://github.com/MicrosoftLearning/MB-500-Microsoft-Dynamics-365-Finance-and-Operations-Apps-Developer/blob/master/Instructions/Labs/Lab%204%20-%20Metadata%20Extension%20%26%20Development.md#task-3-menu-extension-accountsreceivable

**Comment by janeteblake** [comment]
Good feedback; we created a new Task, thank you!
